app/scripts/lib/oauth-client.js

```diff
@@ -49,6 +50,18 @@ define(function (require, exports, module) {
     },
 
     /**
+     *
+     * @param {Object} params
+     * @param {String} params.assertion
+     * @param {String} params.client_id
+     * @param {String} params.scope
+     * @returns {Promise}
+     */
+    getClientKeyData: function getClientKeyData(params) {
```

Comment:
Are there any new errors that need to be added to [oauth-errors.js](https://github.com/mozilla/fxa-content-server/blob/b5ec8810a68517a4d4bc36310598926f9b8f9f53/app/scripts/lib/oauth-errors.js#L90?)?

Reply:
Nope, we added the PKCE errors earlier, so should be good 👍  .Thanks for checking!